feat: added work and about me section

Rophah opened the pull request.

Rophah committed:
feat: added work and about me section
fix

Rophah committed:
Delete test.css

Rophah committed:
fix: fixed all linters errors
fix: added margin to the anchor tag in the about section

Rophah committed:
Fix: Deleted test.css as  it is not needed

Rophah committed:
Merge branch 'mobile' of https://github.com/Rophah/Portfoliopage into mobile

adejam commented:
Hi, while it will be lovely to review this PR, it would be great to have a detailed description of what the PR does or aims to achieve. Also, this project at this stage should be viewable online, there are various tools with which to achieve this and I recommend using [Netlify](netlify.com) as it is easy to use and offers links/deploys for PR preview

Rophah committed:
comment new section

Rophah commented:
Thank you very much for helping me to deploy to neflify.

Jazakunmullahu khearan

On Wed, Jun 1, 2022, 2:43 PM netlify[bot] ***@***.***> wrote:

> ✅ Deploy Preview for *rophah* ready!
> Name Link
> 🔨 Latest commit 4897118
> <https://github.com/Rophah/Portfoliopage/commit/48971183fd7d6365e6adf105dbfab20a62f53da4>
> 🔍 Latest deploy log
> https://app.netlify.com/sites/rophah/deploys/62976d04bb6a66000835c3b7
> 😎 Deploy Preview https://deploy-preview-3--rophah.netlify.app/
> 📱 Preview on mobile Toggle QR Code...
>
> [image: QR Code]
> <https://camo.githubusercontent.com/2ce0962f6f59ec4414d730efdf041dfc27696acecc1b17dce936a3480923caf0/68747470733a2f2f6170702e6e65746c6966792e636f6d2f71722d636f64652f65794a30655841694f694a4b563151694c434a68624763694f694a49557a49314e694a392e65794a31636d77694f694a6f64485277637a6f764c32526c6347787665533177636d5632615756334c544d744c584a7663476868614335755a58527361575a354c6d467763434a392e6d4e4d6874476c686175557568416d435467734a42446d7462434232614a6634727849646e6f744836476b>
>
> *Use your smartphone camera to open QR code link.*
> ------------------------------
>
> *To edit notification comments on pull requests, go to your Netlify site
> settings
> <https://app.netlify.com/sites/rophah/settings/deploys#deploy-notifications>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Rophah/Portfoliopage/pull/3#issuecomment-1143631361>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALFK6VQFYTFZ5EG7GM7KRGTVM5SJJANCNFSM5XMEMJOQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

adejam commented on css/main.css:
```diff
@@ -153,17 +156,220 @@ nav {
+}
+
+.card {
+  display: flex;
```
Please let's avoid the mistake of repetition, we are repeatedly using the same property over and over again. Please not that we should make our code dry as much as possible

Rophah replied:
Okay, I will work on this immediately. Thank you very much

Rophah replied:
Thank you so much for the observation. .card is a class defining rules holding the all the cards. I defined display:flex in different rules because without it other flex properties won't work.

adejam commented on css/main.css:
```diff
@@ -153,17 +156,220 @@ nav {
+  border: 1px solid var(--button);
+  padding: 10px;
+  border-radius: 10px;
+  font-weight: 500;
```
Repeated code again

adejam commented on index.html:
```diff
@@ -84,5 +84,224 @@
+        <p class="connect">Let's connect</p>
+
+        <div class="vectorimages vimage">
+          <img src="images/introduction/Vector-5.svg" class="vector" alt="vector5" />
+          <img src="images/introduction/Vector-3.svg" alt="vector3" />
+          <img src="images/introduction/Vector-2.svg" alt="vector2" class="vector" />
+          <img src="images/introduction/Vector-1.svg" class="vector" alt="vector1" />
+          <img src="images/introduction/Vector-4.svg" class="vector" alt="vector4" />
+          <img src="images/introduction/Vector.svg" alt="vector" />
```
please notes that these are not just images but links to your social and professional accounts. You made the same mistake at the header section of the page please let us add the correct links and styling. We should make the list of icons uses a `ul` i.e(vectorimages class should be `ul` and not `div`) tag to group them, and `li` per icon; each icon has a link (a tag)

Rophah replied:
Okay, noted.

Rophah replied:
Thank you very much, error corrected.

adejam commented on index.html:
```diff
@@ -84,5 +84,224 @@
+    <!-- About me section -->
+    <section class="aboutsection">
+      <div class="aboutparagraph">
+        <h2 class="name">About</h2>
+        <h2 class="name">Myself</h2>
```
This should be just one `h2` tag, like `<h2>About myself</h2>`. It only seemed to be wrapping in the design which means that the text should be wrappable if the screen gets so small that it can't contain all the text

Rophah replied:
Correction will be done. thank you

adejam commented on index.html:
```diff
@@ -84,5 +84,224 @@
+          coded, don’t hestiate to contact me.
+        </p>
+
+        <p class="connect">Let's connect</p>
```
This should be an `hx` tag. If you notice from the design that it looks quite bigger than a normal `p` element. You can use `h5` or `h6` instead and style it accordingly

Rophah replied:
Okay, i will look into this

adejam commented on index.html:
```diff
@@ -84,5 +84,224 @@
+          <div class="arrow">
+            <i class="fa-solid fa-angle-down"></i>
+          </div>
+        </div>
+
+        <div class="languages">
+          <div class="writtenlanguage">
+            <img src="images/worksection/js.png" alt="javaScript" />
+            <p>JavaScript</p>
+          </div>
+
+          <div class="writtenlanguage">
+            <img src="images/worksection/html.png" alt="HTML" />
+            <p>HTML</p>
+          </div>
+
+          <div class="writtenlanguage">
+            <img src="images/worksection/css.png" alt="CSS" />
+            <p>CSS</p>
+          </div>
+        </div>
+
+        <div class="frameworkdiv">
+          <div class="frameworks">
+            <p>Frameworks</p>
+            <div class="arrow">
+              <i class="fa-solid fa-angle-right"></i>
+            </div>
+          </div>
+          <div class="line"></div>
+
+          <div class="frameworks">
+            <p>Skills</p>
+            <div class="arrow">
+              <i class="fa-solid fa-angle-right"></i>
+            </div>
+          </div>
+
+          <div class="line"></div>
+        </div>
```
You got the interpretation of the whole technologies section wrong. Your technologies categories is a list of "Languages", "Frameworks" and "Skills" that should be `li`s grouped in a `ul` tag. then each technology(i.e: Languages)will have it's own subcategory of it own technology in a list (i.e: "JavasScript", "HTML" and "CSS").  Both of these lists should be arranged using flexbox.

Rophah replied:
I will work on all errors immediately. Thank you.

adejam reviewed (changes requested):
# Still requires alot of improvements.

Please look at my comments and work on them as necessary

Rophah commented:
Thank you very much for the comments. I will work on them immediately.
Assalamalaikun warahmotullahi

Rophah committed:
fix: removes the second <h2>
fix: changes the <p> in let's connect to <h5>
fix: changes the technology categories and subcategories to a list using <ul> and <li>
style: removes the font-weight: 500 on line 263 and creates a class of fontweight500 with property 'font-weight: 500' so as to make my code dry

Rophah committed:
fix: runs linter

Rophah committed:
fix: creates <ul> inside the <li> to line 283
fix: creates <ul> inside the <li> to line 302

adejam reviewed (approved):
# Looks good to me

### Goodluck on your next milestones
